common.blocks/prism/prism.js

```diff
@@ -0,0 +1,5 @@
+modules.define('prism', function(provide) {
+    /* borschik:include:../../node_modules/prismjs/prism.js */
```

Comment:
it is still exposed to global scope. is there any possibility to wrap the whole include into IIFE with some args to prevent it to be exposed?

Reply:
No, Prism writes to window and even use it's features in code.

explained in detailed in tg.